cts/schemas/test-3/ref/multiple-location-rules.ref-99

```diff
@@ -80,50 +80,58 @@
         <resource_set id="ban-rsc3-rsc4-node1-node2-set2">
           <resource_ref id="rsc4"/>
         </resource_set>
-        <rule id="ban-rsc3-rsc4-node1-node2-rule1" score="-INFINITY">
-          <expression id="ban-rsc3-rsc4-node1-node2-rule1-expr" attribute="#uname" operation="eq" value="node1"/>
-        </rule>
-        <lifetime>
-          <rule id="ban-rsc3-rsc4-node1-node2-lifetime-rule1">
-            <date_expression id="ban-rsc3-rsc4-node1-node2-lifetime-rule1-expr" operation="in_range" start="2004-001"/>
+        <rule id="pcmk__3_10_upgrade-pcmk__3_10_upgrade-ban-rsc3-rsc4-node1-node2-1-lifetime-and-rule" boolean-op="and">
+          <rule id="ban-rsc3-rsc4-node1-node2-rule1" score="-INFINITY">
+            <expression id="ban-rsc3-rsc4-node1-node2-rule1-expr" attribute="#uname" operation="eq" value="node1"/>
           </rule>
-          <rule id="ban-rsc3-rsc4-node1-node2-lifetime-rule2">
-            <date_expression id="ban-rsc3-rsc4-node1-node2-lifetime-rule2-expr" operation="in_range" end="2005-001"/>
+          <rule id="pcmk__3_10_upgrade-pcmk__3_10_upgrade-ban-rsc3-rsc4-node1-node2-1-lifetime-or-rule" boolean-op="or">
+            <rule id="ban-rsc3-rsc4-node1-node2-lifetime-rule1">
+              <date_expression id="ban-rsc3-rsc4-node1-node2-lifetime-rule1-expr" operation="in_range" start="2004-001"/>
+            </rule>
+            <rule id="ban-rsc3-rsc4-node1-node2-lifetime-rule2">
+              <date_expression id="ban-rsc3-rsc4-node1-node2-lifetime-rule2-expr" operation="in_range" end="2005-001"/>
+            </rule>
```

Comment:
Note that every date is after 2004-001 or before 2005-001. Doesn't matter for the purposes of the schema transform though.

Reply:
Yeah I wasn't concerned; a lot of these schema transformation tests use pretty contrived configurations anyway, and I'm more concerned about structural changes. Happy to modify some of the dates if you decide you want that.